Compiler/BackEnd/EvaluateParameter.mo

```diff
@@ -892,7 +908,21 @@ algorithm
       // apply replacements
       (e, true) = BackendVarTransform.replaceExp(e, iReplEvaluate, NONE());
       (e, _) = ExpressionSimplify.simplify(e);
-      e = EvaluateFunctions.evaluateConstantFunctionCallExp(e, FCore.getFunctionTree(iCache), Flags.getConfigBool(Flags.EVAL_CONST_ARGS_ONLY));
+      // If call with constant arguments then evaluate
```

Comment:
As far as I have seen by now ceval evaluates more than evaluateConstantFunctionCallExp...
Additionally I have some models which take a very long time to evaluate the bindings with evaluateConstantFunctionCallExp but do not come to a result anyway.
For example this is one testrun I did before the commit:
`
evaluateConstantFunctionCallExp:
===============================
timeFrontend = 38.557968424,
timeBackend = 175.138638479,
timeSimCode = 255.884944745,
timeTemplates = 0.822031247,
timeCompile = 1.607657314

CEVAL:
=====
timeFrontend = 38.855595454,
timeBackend = 4.27404939,
timeSimCode = 1.014362718,
timeTemplates = 0.8161275649999999,
timeCompile = 1.637323581
`
I will investigate how many functions are actually evaluated with both variants.